eng/Version.Details.xml

```diff
@@ -1,12 +1,8 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Diagnostics.Monitoring" Version="5.0.328102">
```

Comment:
Will this cause any issues with darc for release build prep?

Reply:
I don't think it will.